holdings_maintenance/pds4/pds4linkshelf.py

```
@@ -0,0 +1,1219 @@
+    dirpath = os.path.abspath(dirpath)
+    pdsdir = pdsfile.Pds4File.from_abspath(dirpath)
+
+    dirpath_ = dirpath.rstrip('/') + '/'
+
+    logger = logger or pdslogger.PdsLogger.get_logger(LOGNAME)
+    logger.replace_root(pdsdir.root_)
+    logger.open('Reading link shelf file for', dirpath, limits=limits)
+
+    try:
+        (link_path, lskip) = pdsdir.shelf_path_and_lskip('link')
+        prefix_ = pdsdir.bundle_abspath() + '/'
```

Flake8 (7.2.0)

[error] local variable 'prefix_' is assigned to but never used

(F841)